packages/falso/src/lib/movie-character.json

```diff
@@ -0,0 +1,24 @@
+{
+  "data": [
```

Comment:
Could you scrape the list from here, please?

https://www.empireonline.com/movies/features/100-greatest-movie-characters/

Reply:
Sure. Will do.

Reply:
Done.